Code/Editor/WelcomeScreen/WelcomeScreenDialog.cpp

```diff
@@ -325,7 +325,14 @@ void WelcomeScreenDialog::OnNewLevelBtnClicked([[maybe_unused]] bool checked)
 
 void WelcomeScreenDialog::OnNewLevelLabelClicked([[maybe_unused]] const QString& path)
```

Comment:
The path is now used, so you should remove `[[maybe_unused]]`.

Reply:
done.